pallets/capacity/src/lib.rs

```diff
@@ -82,8 +85,39 @@ pub mod pallet {
 
 		/// Function that allows a balance to be locked.
 		type Currency: LockableCurrency<Self::AccountId, Moment = Self::BlockNumber>;
+
+		/// The minimum required token amount to stake. It facilitates cleaning dust when unstaking.
+		#[pallet::constant]
+		type MinimumStakingAmount: Get<BalanceOf<Self>>;
+
+		/// The maximum number of `unlocking` chunks a `StakingAccountLedger` can have. It determines how many unique Epochs a Staker may be unstaked in.
```

Comment:
Should this state something like:
/// The maximum number of `unlocking` chunks a `StakingAccountLedger` can have. It determines how many concurrent unstaked chunks may exist.